jenkins/config_backup/config_backup_playbook.yml

```diff
@@ -34,6 +35,16 @@
         dest: "{{ config_backup_dir }}/tiaas/"
       with_items:
         - db.sqlite3
+    - name: Copy list of venv packages
+      block:
+      - name: Get list of packages
+        command: /mnt/galaxy/venv/bin/pip list --format=freeze
+        register: pip_packages
+      - name: Save package list locally
+        local_action:
```

Comment:
@cat-bro optionally, you can use `delegate_to: localhost` instead of the older form `local_action`. They are both equivalent, so it's ok as is.